.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - master
+    - _ci_test
```

Comment:
Why this change? I thought we should run CI for all PRs which target the `master` branch?

Reply:
The CI runs always when a PR is opened with target mhammond:master - but not before, when a branch is only pushed to github. (At least for forks on github). See the feature branches of any forks on github ("View all branches"): no CI actions symbol on the top commits (unless it contains this kind of change in the main.yml).

But often for non-trivial changes one wants to trigger the CI before opening a PR. So when I wanted this, in the past I added another temporary commit which added the feature branch name to the "main.yml/on:push:branches:" - which is laborious.
When "_ci_test" is pre-added generally, (any)one can simply make _ci_test an alias branch for the current feature branch (or any commit) of interest and push it to github to trigger the CI. No need for in & out of temporary commits.

Reply:
Fair enough - how about a summary of the above as a comment?

Reply:
could be announced like this:
```
--- a/README.md
+++ b/README.md
@@ -29,7 +29,8 @@ A special shout-out to @xoviat who provided enormous help with the github move!
 
 Feel free to [open issues](https://github.com/mhammond/pywin32/issues) for
 all bugs (or suspected bugs) in pywin32. [pull-requests](https://github.com/mhammond/pywin32/pulls)
-for all bugs or features are also welcome.
+for all bugs or features are also welcome. Note: A CI test run can be triggered before
+opening a PR by pushing a feature branch as (alias) branch "_ci_test" to github.
 
 However, please **do not open github issues for general support requests**, or
 for problems or questions using the modules in this package - they will be

```

( Activating the CI for all and any branch pushes may be too burdensome for github.? )

Reply:
> ( Activating the CI for all and any branch pushes may be too burdensome for github.? )

Actually, I doubt it would, so I'd be fine with that.

Reply:
I removed that commit from this PR so far.  (Same was in the py3.10 draft PR.) I may try a on:push:branches:* or whatever when needing it next time. When I remember right, this would trigger the CI run on github twice then for any normal push+PR open despite idential SHA.

Reply:
> But often for non-trivial changes one wants to trigger the CI before opening a PR.

In this situation with other repos, I typically just open a PR anyway, marking it as draft. It looks like wildcards will probably work, but not for forks?  (But yeah, I'm not too bothered about chewing github resources, so I'm fine with "every branch for every fork" until they complain in some way)